src/components/About.tsx

```
@@ -0,0 +1,71 @@
+      <div className="container flex items-center justify-center">
+        <div className="w-full max-w-5xl px-4 md:px-6">
+          <div className="grid gap-8 lg:grid-cols-[2fr_1fr] lg:gap-12 items-start text-left">
+            <div className="flex flex-col justify-center space-y-6">
+              <div className="space-y-2">
+                <h2 className="text-3xl font-bold tracking-tighter sm:text-4xl md:text-5xl">About Me</h2>
+                <p className="text-gray-500 dark:text-gray-400 text-lg leading-relaxed">
+                I specialize in developing scalable software solutions, interactive web applications, 
+                and robust data pipelines for complex biological analyses. With over 7 years of experience spanning full-stack web development, 
+                relational database optimization, and leveraging large language models for advanced data aggregation, my work consistently 
+                bridges the gap between bioinformatics innovation and practical applications in genomic research and drug discovery. 
+                I'm passionate about creating tools that not only streamline scientific workflows but also empower researchers to explore 
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)